server/api/routers/movies.js

```diff
@@ -25,38 +25,46 @@ movieRouter.post('/imdbsearch', async (req, res) => {
 
 movieRouter.post('/order', async (req, res) => {
   const { id } = req.body;
-  const movie = await imdb.get({ id }, { apiKey: process.env.IMDB_API_KEY });
-  const newMovie = await Movie.create({
-    title: movie.title,
-    director: movie.director.split(', '),
-    actors: movie.actors.split(', '),
-    awards: movie.awards,
-    boxoffice: movie.boxoffice,
-    genres: movie.genres.split(', '),
-    id: movie.imdbid,
-    metascore: movie.metascore,
-    plot: movie.plot,
-    poster: movie.poster,
-    rated: movie.rated,
-    rating: movie.rating,
-    production: movie.production,
-    released: movie.released,
-    runtime: movie.runtime,
-    writer: movie.writer.split(', '),
-    year: movie.year,
-    price: 0.99,
-    stock: 50,
-  });
+  const checkMovie = await Movie.findAll({ where: { id } });
 
-  if (newMovie) {
-    res.send(newMovie);
+  if (checkMovie) {
+    res.sendStatus(204);
```

Comment:
204 is "no content" is that what we want to send?

Reply:
anything in the 400's seems to bring the page to a halt, this allows us to throw up a toast notification